js/screens/Onboarding/Onboarding.js

```diff
@@ -112,4 +113,8 @@ const OnboardingPage = ({ navigation }) => {
   );
 };
 
+OnboardingPage.propTypes = {
```

Comment:
You are missing propType validation for some of your props. And make one PR for all of your propTypes.